### PR TITLE
191 Add thumbnail to ItemDisplay and SongPreview

### PR DESCRIPTION
**Pull request by Xenocidel**

### CODE REVIEW

File: S2VX.Game/SongSelection/Containers/SongPreview.cs [hunk 1]
```diff
@@ -15,10 +16,11 @@ public class SongPreview : CompositeDrawable {
         [Resolved]
         private ScreenStack Screens { get; set; }
 
-        public string CurSelectionPath { get; set; }                           // Set in SongSelectionScreen
-        public string StoryPath { get; set; }                                  // Set in SongSelectionScreen
-        public string AudioPath { get; set; }                                  // Set in SongSelectionScreen
-        public StorageBackedResourceStore CurLevelResourceStore { get; set; }  // Set in SongSelectionScreen
+        public string CurSelectionPath { get; set; }                           // Must be set in SongSelectionScreen
```
Comment (maxrchung) [inline S2VX.Game/SongSelection/Containers/SongPreview.cs:19]: Why don't you just set these in a constructor since they are required?

Comment (Xenocidel) [reply S2VX.Game/SongSelection/Containers/SongPreview.cs:19]: Will do

**Review by Naranja-Sagged — APPROVED** [review]

File: S2VX.Game/SongSelection/Containers/SongPreview.cs (hunk 1 above)
Comment (maxrchung) [reply S2VX.Game/SongSelection/Containers/SongPreview.cs]: I did a quick Ctrl+F, and I think Border still has something with the `// Set in SongSelectionScreen` comment. Should update this too?

Comment (maxrchung) [reply S2VX.Game/SongSelection/Containers/SongPreview.cs]: Bumping in case you missed this

Comment (Xenocidel) [reply S2VX.Game/SongSelection/Containers/SongPreview.cs]: Ah only Ctrl+F'd for "must". Thanks for catching this, my bad for missing. Updated.

**Review by maxrchung — APPROVED** [review]

**Pull request merged**